ci/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN rustup component add rustfmt llvm-tools-preview clippy
 
 # install build tools
 RUN cargo install cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-sort cargo-make cargo-cache \
-    && cargo install --git https://github.com/risinglightdb/sqllogictest-rs --bin sqllogictest \
+    && cargo install --git https://github.com/risinglightdb/sqllogictest-rs --rev 65b122f --bin sqllogictest \
```

Comment:
I'd like to specify the revision to make the bump of sqllogictest more explicit. 😄 It's the lastest revision on `main` now. cc @xxchan

Reply:
very good